manifests/params.pp

```diff
@@ -42,4 +46,10 @@
   } else {
     $configure_epel = false
   }
+
+  if $::osfamily == 'OpenBSD' {
```

Comment:
can you use a selector here?
https://puppet.com/docs/puppet/5.3/lang_conditional.html#selectors